### PR TITLE
Portfolio: setup and mobile first.

### PR DESCRIPTION
**Pull request by samif1**
In this projects:
    1) index.html file are modified by adding       another 3 section tag
    2) style.css file are modified by for the styling purpose of the modified index.html file
   3) images folder is also modified by adding another resources.

### CODE REVIEW

**Comment by Gopxfs** [comment]
# STATUS: INVALID REQUEST

Hi @samif1 👋,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project that doesn't contain a professional Pull Request. Check [this link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to understand more about it. Please notice that your PR title is different than the exercise you requested a review.

Your Code Review Request will be marked invalid in your Dashboard, so please submit a new one once ready 🙏.
Invalid requests don't count as a review, so don't worry about that 😄!

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -63,5 +63,248 @@ <h1 class="samuel">I’m Samuel Glad to see you!</h1>
             </div>
         </div>
     </section>
+    <section class="works">
+        <div class="card-works">
+          <img
+            src="images/Snapshoot Portfolio1.svg"
+            alt="Snapshoot Portfolio"
+          />
```
Comment (tufoinnkuo10) [inline index.html:71]: - [ ] kindly use the correct background color like the figma version .. `#F7F7F9;`

yours
Screenshot (568)_LI
figma
Screenshot (567)_LI

File: index.html [hunk 2]
```diff
@@ -63,5 +63,248 @@ <h1 class="samuel">I’m Samuel Glad to see you!</h1>
...
+      </section>
+  
+      <section class="about">
+        <h2 class="primary-text">About Myself</h2>
+        <p class="secondary-test">
+          Hello I’m a software developer! I can help you build a product , feature
+          or website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <p class="supportive-text">Let's Connect</p>
```
Comment (tufoinnkuo10) [inline index.html:180]: - [ ] kindly use the correct color on the` lets connect `heading so it follows the figma design
- [ ] lets connect is a heading so it should be in an `hx tag`
- [ ] you seem to be missing the social icons. please implement that

yours
Screenshot (570)_LI
figma
Screenshot (569)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
## Hi @samif1 

There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

## Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🏆 🥇 🍹 

Hi @samif1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉  🎉 🎉

approvedstamp

## Highlights 💯 
- No Linter errors ✔️ 
- Professional Readme file ✔️ 
- Descriptive commit messages ✔️ 
- Github flow used ✔️ 
- Descriptive Pull request ✔️ 

## Optional suggestions

- I would like to kindly suggest that you name your branch after a current feature you are to implement (e.g "setup-navbar", "add-form", "about-me-section" e.t.c) instead of using the name `feature` as your branch name 😉 🌸 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**